package.json

```diff
@@ -25,10 +25,9 @@
     "test": "jest app/javascript/ bin/ --coverage",
     "test:watch": "jest app/javascript/ bin/ --watch",
     "postcss": "postcss public/assets/*.css -d public/assets 2> postcss_error.log",
-    "e2e": "cypress open --config-file cypress.dev.json",
-    "e2e:runner": "bin/e2e",
-    "e2e:server": "bin/rails s -p 3000 -b 0.0.0.0",
-    "e2e:ci": "start-server-and-test e2e:server http://localhost:3000 e2e:runner"
+    "pree2e": "RAILS_ENV=test bin/e2e-setup",
```

Comment:
Now there is a pre step to setup the environment for e2e tests when running things locally.